source/jormungandr/tests/check_utils.py

```diff
@@ -540,6 +540,59 @@ def is_valid_graphical_isochrone(isochrone, tester, query):
         is_valid_feed_publisher(feed_publisher)
 
 
+def is_valid_single_coord(coord, is_valid_type, name):
+    min_type = get_not_null(coord, 'min_' + name)
+    middle_type = get_not_null(coord, 'middle_' + name)
+    max_type = get_not_null(coord, 'max_' + name)
```

Comment:
not sure about the redundancy max n = min n+1 and middle = (max+min)/2 but this is usefull for integration, this is a O(n) impact so... Unless you want to check the size with only min of each cell and the max of the matrix to see what is the best case size I guess we can go with that untill we are flooded with heat map requests.